docs/2.develop/1.quickstart.md

```diff
@@ -0,0 +1,112 @@
+
+---
+
+## Prerequisites
+In order to successfully follow this quickstart guide you will need Node.js and RUST.
+
+### 🚀 Node.js
+Download and install [Node.js](https://nodejs.org/en/download/). Then, install `yarn`: `npm install --global yarn`.
+
+### 🦀 Rust and Wasm
+
+Follow [these instructions](https://doc.rust-lang.org/book/ch01-01-installation.html) for setting up Rust. Then, add the `wasm32-unknown-unknown` toolchain which enables compiling Rust to [Web Assembly (wasm)](https://webassembly.org/), the low-level language used by the NEAR platform.
+
+#### Linux and MacOS:
+
+```bash
+# Get rust
+curl --proto '=https' --tlsv1.2 https://sh.rustup.rs -sSf | sh
+source $HOME/.cargo/env
+
+# Add the wasm toolchain
+rustup target add wasm32-unknown-unknown
+```
+
+---
+
+## 👋 Hello NEAR!
+
+Lets create your first decentralized app using NEAR, for this you will need just one command.
+Using [npm's npx](https://blog.npmjs.org/post/162869356040/introducing-npx-an-npm-package-runner):
+
+    npx create-near-app [options] hello-near
+
+**or**, if you prefer [yarn](https://classic.yarnpkg.com/en/docs/cli/create/):
+
+    yarn create near-app [options] hello-near
+
+Without any options, this will create a project with a **vanilla JavaScript** frontend and an AssemblyScript smart contract. If you prefer, you can create a **React** frontend or a **Rust** smart contract you can use these options:
+
+* `--frontend=react` – use [React](https://reactjs.org/) for your frontend template
+* `--contract=rust` – use [Rust](https://docs.near.org/docs/develop/contracts/rust/near-sdk-rs) for your smart contract
+
+---
+
+## Interacting With your dApp
+
+Before we talk about the structure in the folder structure of your dApp, go ahead and try it!. For this, 
+enter the directory and use `yarn` to start it.
+
+```bash
+cd hello-near
+yarn dev
+```
+
+Your contract will then be **compiled** and **deployed** to an **account** in the `testnet` network. When done, a browser window should open with your dApp. Go ahead, sign in and play around a bit, once you get familiarized with the application come back.
+
+![img](./assets/hello-near.png)
+*Frontend of your first decentralized app in NEAR*
+
+---
+
+## Structure of your dApp
+
+Let's take a closer look to the structure of your `hello-near` dApp:
+
+1. The frontend code lives in the `/src` folder. Note that `/src/index.html` loads `/src/index.js`, which deals with connecting to the contract.
+2. The smart contract code is in the `/contract` folder. The content differs from RUST to Assemblyscript, so check the README there for more info.
+3. The compiled smart contract can be found in `/out/main.wasm`.
+4. The account's name in which the contract was deployed is in `/neardev/dev-account`.
+
+---
+
+## Testing
+
+When writing smart contracts it is very important to test all methods exhaustively. In this
```

Comment:
Are readers here both experienced and new developers? In case we have only experienced devs here then making this section shorter with just the first paragraph or so with the reference to the Testing section would be enough?